client/post-editor/editor-drawer/index.jsx

```diff
@@ -113,8 +114,9 @@ var EditorDrawer = React.createClass( {
 	},
 
 	renderSharing: function() {
+		const currentUser = user.get();
 		return (
-			<EditorSharingContainer site={ this.props.site } />
+			<EditorSharingContainer site={ this.props.site } currentUserID={ currentUser.ID } />
```

Comment:
Though highly unlikely ( maybe not even possible ? ) do we need to consider the possibility of not having a `currentUser` here and return `null` instead?